### PR TITLE
Add Onfido sandbox to selfie prototype

### PR DESCRIPTION
**Pull request by merlinc**
## Proposed changes

Use Onfido API with GDS styling to perform a document and likeness check.

Note: Although there are a lot of files in this PR, they're mostly localisation and page setup, the form framework handles most of the work, so there is very little application code to be reviewed.

### What changed

- Added an Onfido sub-router & form-wizard
- Added "clear" path through Onfido  system
- Fixed uploading files using busboy
- Return signed packet of Onfido data to selfie endpoint
   

### Why did it change

- We want to test out the Onfido API
- 

### Issue tracking

- [PYI-179](https://govukverify.atlassian.net/browse/PYI-179)

### Environment variables or secrets

- [x] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

### CODE REVIEW

File: views/onfido/breakdown.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{% extends "form-template.njk" %}
```
Comment (merlinc) [inline views/onfido/breakdown.html:1]: This repetitive boilerplate shouldn't be necessary - I consider this a bug in `hmpo-app` / `hmpo-components` to be fixed.

File: views/onfido/document-upload.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+<h1>{{ hmpoTitle }}</h1>
+
+<ul class="govuk-list govuk-list--bullet">
+    <li>This page is used to represent what a user would see</li>
+    <li>Onfido in Sandbox mode will not process this image at all</li>
+    <li>Do not upload personal documents to the sandbox environment</li>
+</ul>
+
+<form
+        enctype="multipart/form-data"
```
Comment (merlinc) [inline views/onfido/document-upload.html:16]: `hmpo-forms` doesn't allow changing of the `enctype` so it has to be specified here.

To be fair, on upload pages, there is usually a lot of bespoke content shaping to ensure users understand what is happening, which means the automatic `hmpo-app` approach doesn't work as well here.

File: onfido/controllers/onfido-check-status.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
+const BaseController = require("hmpo-form-wizard").Controller;
+const { Onfido, Region } = require("@onfido/api");
+const debug = require("debug")("selfie:onfido-check");
+
+const onfido = new Onfido({
```
Comment (merlinc) [inline onfido/controllers/onfido-check-status.js:5]: Also, this SDK should be instantiated once on app startup, and then reused as needed within each controller - not instantiated every time a controller needs it.

**Review by martin-bucinskas — APPROVED** [review]
LGTM

**Review by martin-bucinskas — APPROVED** [review]
LGTM

**Pull request merged**